lib/store_model/ext/active_model/attributes.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "store_model/ext/parent_assignment"
+
+module StoreModel
+  # ActiveModel::Attributes patch with parent tracking support
+  module Attributes
+    include ParentAssignment
+
+    private
+
+    def attribute(*)
```

Comment:
Do you know the case when we need this patch? I've tried to remove it and all specs are passing, looks like this scenario is covered by `#_read_attribute` method

Reply:
Good catch! We had a preemptive parent assignment on attribute write in specs :)
Now couple specs should fail accordingly